webui/react/src/components/OverviewStats.tsx

```diff
@@ -24,7 +24,9 @@ const OverviewStats: React.FC<Props> = (props: Props) => {
       </Row>
       <Row width="fill">
         <Label strong truncate={{ tooltip: true }}>
-          {props.children}
+          <span style={{ color: props.onClick ? 'var(--theme-status-active)' : 'inherit' }}>
+            {props.children}
+          </span>
```

Comment:
does `Link` not work?

Reply:
https://github.com/determined-ai/determined/blob/e956f286e2effc466dbc3c998257839d22f4ab3f/webui/react/src/components/OverviewStats.tsx#L17-L32

it works in terms of the appearance, but that might cause nested anchor tag (`<a>`) which is forbidden. 

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element
> there must be no [interactive content](https://html.spec.whatwg.org/multipage/dom.html#interactive-content-2) descendant

Reply:
honestly it's kind of weird we have a link for a specific line within a clickable card in the first place, but I'd prefer to use `Link`, partially because it also has the hover styling. 

I think the nested `<a>` is only happening because the card content is also unnecessarily wrapped in a `Link` when instead it should be using the `Card`'s `onClick` (which also adds hover styling to the card itself)

suggestion: 

```
   return (
    <Card onClick={props.onClick}>
      <Column>
        <Row>
          <Label size={TypographySize.XS} truncate={{ tooltip: true }}>
            {props.title}
          </Label>
        </Row>
        <Row width="fill">
          <Label strong truncate={{ tooltip: true }}>
            {props.onClick ? (
              <Link>
                {props.children}
              </Link>
            ) : props.children}
          </Label>
        </Row>
      </Column>
    </Card>
  );
```

Reply:
I think we should probably still have `onClick` on `Card`?

Reply:
imo only blue text should be clickable.

Reply:
if product agrees, sure. my suggestion was to avoid removing functionality.

Reply:
product agreed with only text link ([Slack](https://hpe-aiatscale.slack.com/archives/C06G9QEKUJZ/p1721336534801819))